README.md

```diff
@@ -374,6 +374,7 @@ There are also many architecture libraries in the Swift and iOS community. Each
 * [RxFeedback](https://github.com/NoTests/RxFeedback.swift)
 * [Mobius.swift](https://github.com/spotify/mobius.swift)
 * [ReactiveFeedback](https://github.com/babylonhealth/ReactiveFeedback)
+* [Reactor](https://github.com/ReactorSwift/Reactor)
```

Comment:
Could you move this up to up to be between RIBs and ReSwift? we're ordering these by title length, just to be arbitrary